core/gather/gatherers/trace-elements.js

```diff
@@ -231,7 +231,7 @@ class TraceElements extends FRGatherer {
     }
 
     // These should exist, but trace types are loose.
-    const lcpData = processedNavigation.largestContentfulPaintEvt?.args?.data;
```

Comment:
![emperorsnewgroove](https://user-images.githubusercontent.com/316891/213362484-2faa868d-1983-4062-82ae-8483ab6fbbb5.gif)

Reply:
wait, so is this a bug fix? we'll now identify the LCP element as being inside an iframe?

Reply:
> wait, so is this a bug fix? we'll now be identifying the LCP element as being inside an iframe, whereas before we never did?

Yeah, unfortunately a definite bug since #14344. Fortunately only triggered if the page's LCP was an image in an iframe but the main frame's LCP was text. Not a common occurrence but not great :/